lib/sipper/cli.ex

```diff
@@ -23,6 +23,8 @@ defmodule Sipper.CLI do
       switches: [
         max: :integer,
         dir: :string,
+        ignore: :string,
+        asc: :boolean
```

Comment:
Would you mind putting a trailing comma on this one as well? It's [my preferred style](https://github.com/barsoom/devbook/tree/master/styleguide#add-commas-to-the-end-of-multiline-lists-and-hashes).